### PR TITLE
A11y home

### PR DESCRIPTION
**Pull request by lc512k**
@wheresrhys Moved a11y home to static 

We'll need to keep the github repo as all a11y issues are logged there, but this will allow us to delete the heroku app, etc

- added pa11y to this repo
- fixed a11y on the more about ft page
- changed the port to 3002 for `make a11y` to run locally
- added logic to turn off flags if needed

### CODE REVIEW

**Comment by lc512k** [comment]
relates to https://github.com/Financial-Times/next-service-registry/pull/240

**Comment by leggsimon** [comment]
For some reason, `make install` seems to be running pa11y. I don't know whether this is to do with the updated n.Makefile but it seems to be testing it before it ever builds the app and tries to float it on heroku. I assume that's not meant to happen?

File: server/app.js [hunk 1]
```diff
@@ -17,17 +18,32 @@ app.get('/static/:page', (req, res) => {
 		if (err) return res.sendStatus(404)
 		const data = require(path.join(process.cwd(), 'data', req.params.page + '.json'));
 
+		if (data.disableFlags) {
```
Comment (leggsimon) [inline server/app.js]: I like this idea buuuut, could you document this in the README please?

File: server/app.js [hunk 2]
```diff
@@ -17,17 +18,32 @@ app.get('/static/:page', (req, res) => {
...
+			})
+		}
+
 		const payload = {
 			layout: 'wrapper',
 			title: data.title,
 			data
 		}
+
+		if (data.trackablePageName) {
```
Comment (leggsimon) [inline server/app.js]: This as well

File: server/app.js [hunk 3]
```diff
@@ -17,17 +18,32 @@ app.get('/static/:page', (req, res) => {
...
+		}
+
 		res.render(req.params.page, payload)
 
 	})
 });
 
-const listen = app.listen(process.env.PORT || 3001);
+app.get('/accessibility', (req, res) => {
+	res.redirect('static/accessibility');
```
Comment (leggsimon) [inline server/app.js]: Is the final url meant to be `ft.com/accessibility`? If so then will this work, it'll redirect to `/static/accessibility` but will the vanity url work (assuming `/accessibility` is a vanity url

Comment (lc512k) [inline server/app.js]: I don't really mind the final URL, just want to make sure the page is still reachable at `ft.com/accessibility` (where it lives now in prod). Could do with a vanity url, I guess?

Comment (leggsimon) [inline server/app.js]: In that case it's goooooood

**Pull request merged**